### ISSUE TITLE
When is this repository going to be removed?

### ISSUE THREAD

**Issue by RickKimball** [energia/Energia#922]
This repository is no longer the current Energia source. Should we tell people?

**Comment by robertinant** [comment]
Yes, I am planning to move the repository and then close it down. The EnergiaNG repository will then replace this one. I'll probably keep this repository name but then have it be empty with a README.md that points to the new repo..
